howso/react.amlg

```diff
@@ -30,6 +23,7 @@
 			;{type "list" values "string"}
 			post_process_features (list)
 			;{type "list" values {type "list"}}
+			;see #react for description. if specified must be either lenght of 1 or num_reacts.
```

Comment:
I see it isn't your typo originally, but lets fix it now I guess

Reply:
Not apposed to fixes like that and should be in next commit.